packages/flutter_tools/lib/src/dart/dependencies.dart

```diff
@@ -2,89 +2,49 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:convert';
+import 'package:analyzer/analyzer.dart';
```

Comment:
@goderbauer I was mostly concerned about bloating the initial download size but if we already depend on it, no big deal.